src/components/cms/Dashboard/MyRatings.js

```diff
@@ -77,76 +77,78 @@ class MyRatings extends Component {
           <TableSleleton />
         ) : (
           <TableWrap>
-            <MaterialTable
-              title="My Rating"
-              columns={[
-                {
-                  title: 'Skill Name',
-                  field: 'skillName',
-                  render: rowData => {
-                    return (
-                      <Link
-                        to={{
-                          pathname: `/${rowData.group}/${getSkillFromRating(
-                            rowData.skillName,
-                          )}/${rowData.language}`,
-                        }}
-                      >
-                        {(
-                          rowData.skillName.charAt(0).toUpperCase() +
-                          rowData.skillName.slice(1)
-                        ).replace(/[_-]/g, ' ')}
-                      </Link>
-                    );
+            {ratingsData.length !== 0 && (
```

Comment:
Please add a  safety check here `ratingsData && Array.isArray(ratingsData) && ratingsData.length`

Reply:
Ok then, doing the same changes to `Skills` and `Analytics` as well